proto/sqlblobs/sqlblobs.proto

```diff
@@ -178,30 +178,4 @@ message RequestCancelInfo {
     int64 version = 1;
     int64 initiatedEventBatchID = 2;
     string cancelRequestID = 3;
-}
-
-message TimerInfo {
-    int64 version = 1;
-    int64 startedID = 2;
-    int64 expiryTimeNanos = 3;
-    // taskID is a misleading variable, it actually serves
-    // the purpose of indicating whether a timer task is
-    // generated for this timer info
-    int64 taskID = 4;
-}
-
-message TaskInfo {
```

Comment:
Task/TaskListInfo arent used in this PR but already handled the proto updates for them